### ISSUE TITLE
Efficacy of Skype meetings

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#1302]
As raised by @arojascastro in #1296, the fact that we use Skype for our monthly meetings means that as the team grows, those meetings become more difficult. It raises questions about whether or not those meetings continue to be important, and if so who should attend, and what if any changes should we explore to the way that we hold them.

For those of us who do attend regularly, it is clear that some members of the board have chosen not to contribute to the meetings, and as long as they can fill their roles, I think we have to respect that choice.

**Comment by drjwbaker** [comment]
Those who need to attend should be defined by role profiles that emerge from https://github.com/programminghistorian/jekyll/issues/1285, no?

**Comment by arojascastro** [comment]
I would like to clarify that I think our coordinator's work has been amazing. However, I do believe that since our team is growing quickly and members live in different time zones sometimes it is difficult to find a slot convenient for everybody and gather all together. Once we have defined roles perhaps we can organize smaller team meetings.

**Comment by mdlincoln** [comment]
On 2019-05-31 meeting, we discussed the possibility of creating a staggered call schedule where the general editorial team call occurs every 2 months, and the sub-teams meet on the alternate months. This will reduce the total number of calls everyone needs to participate in.

We won't be able to shift to this schedule until more of the sub-teams are formed, but we suggested aiming to start this schedule in September 2019 once more sub-teams have been formed.
